lib/ioki/model/platform/calculated_point.rb

```diff
@@ -22,7 +22,6 @@ class CalculatedPoint < Base
         attribute :waypoint_type,        on: :read, type: :string
         attribute :walking_duration,     on: :read, type: :integer
         attribute :walking_track,        on: :read, type: :string
-        attribute :station,              on: :read, type: :object, class_name: 'Station'
```

Comment:
The Platform API docs say that we do have a station in there (https://man.pages.ioki.com/api_docs/refs/master/platform-api.html#tag/Ride/paths/~1api~1platform~1products~1{product_id}~1rides/get). Is there a mismatch between docs and reality?

Reply:
Alright, good to know, since we rely on some of those nested things in Schalter, so we'd need to double check (at least for Passenger API)

Reply:
Yes, @rmehner, you should check if you rely on the nested model. 

@ans-ioki When will those API changes go live? Does it not make sense to wait until then?

Reply:
We do for Passenger API (booking & station at least), @ans-ioki does this mean we need to adapt this for us as well?

Reply:
There is only the station_id now.